securedrop/Dockerfile

```diff
@@ -56,6 +58,11 @@ RUN pip install -r requirements/securedrop-app-code-requirements.txt && \
     pip install -r requirements/test-requirements.txt
 
 RUN if test $USER_NAME != root ; then useradd --no-create-home --home-dir /tmp --uid $USER_ID $USER_NAME && echo "$USER_NAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers ; fi
+RUN cp -r /root/.local /tmp/ && chmod +x /tmp/.local/tbb/tor-browser_en-US/Browser/firefox && chmod -R 777 /tmp/.local
+RUN chown -R $USER_NAME.$USER_NAME /tmp/.local/
+
+RUN wget -O /opt/torsocks_2.2.0-2_amd64.deb http://mirrors.kernel.org/ubuntu/pool/universe/t/torsocks/torsocks_2.2.0-2_amd64.deb
+RUN dpkg -i /opt/torsocks_2.2.0-2_amd64.deb && apt-get install -f
```

Comment:
The strategy for installing a more recent version of `torsocks` fetches the package over HTTP and installs whatever it gets, with no checksum or signature verification beforehand. That's bad. Perhaps we can sidestep the need to fetch a newer version of `torsocks` entirely if we instead use `nc` to pass the connection off to tor. Whatever version of `nc` is in the Trusty repos should be sufficient to pull that off!

Reply:
@conorsch Can you please help me with that patch? In my mind I would prefer to rebuild and store a version of `torsocks` somewhere and then pull and use that.